mcpwm_foc.c

```diff
@@ -3823,10 +3866,6 @@ static void control_current(volatile motor_all_state_t *motor, float dt) {
 	float mod_beta  = c * state_m->mod_q + s * state_m->mod_d;
 
 	update_valpha_vbeta(motor, mod_alpha, mod_beta);
-    
-    // Dead time compensated values for vd and vq. Note that these are not used to control the switching times. 
-	state_m->vd = c * motor->m_motor_state.v_alpha + s * motor->m_motor_state.v_beta;
```

Comment:
Is it okay to remove this? The variables are never used, and clobbering the variable which is actually sent downstream for controlling the PWM, in favor of this one, seems like the wrong thing to do.

If it's necessary to keep this `vd` around, it might be best to give it another name.

Reply:
It might be better to rename the other one to vd_control or this one to vd_measured. The measured value is used in vesc tool and the detection code, where measuring it will give slightly better results.

Is the set voltage better to use for FRF than the measured one?

Reply:
> Is the set voltage better to use for FRF than the measured one?

Yes, it is essential for the calculation of the loops that it is the setpoint itself.

Reply:
I'll rename the original to `vd_control` and the other to `vd_measured`.